web/packages/teleport/src/Integrations/Enroll/AwsConsole/IamIntegration/IamIntegration.tsx

```diff
@@ -277,19 +315,74 @@ export function IamIntegration() {
                   />
                 </Flex>
               </CollapsibleInfoSection>
-              <ButtonPrimary
-                width="200px"
-                disabled={phase !== Phase.One}
-                onClick={() => generateCommand(validator)}
-              >
-                Generate Command
-              </ButtonPrimary>
+              {validateName.status === 'error' && validateName.error && (
+                <Danger details={validateName.error.message}>
+                  Error: {validateName.error.name}
+                </Danger>
+              )}
+              {phase === Phase.One ? (
```

Comment:
@kimlisa [wrt#3b](https://github.com/gravitational/teleport/pull/58096#pullrequestreview-3163698380): Per design, the buttons are now primary when the current phase matches, and they change to secondary buttons once progressing to the next step